src/packages/sdk/src/cli/getSchema.ts

```diff
@@ -223,18 +224,28 @@ export async function getSchemaDir(
   }
 
   const schemaPath = await getSchemaPath(schemaPathFromArgs)
-  if (schemaPath) {
-    return path.dirname(schemaPath)
+
+  if (!schemaPath) {
+    return null
   }
 
-  return null
+  return path.dirname(schemaPath)
 }
 
+// TODO: This should probably return string | null to stay consistent with the other functions
 export async function getSchema(schemaPathFromArgs?: string): Promise<string> {
   const schemaPath = await getSchemaPath(schemaPathFromArgs)
 
   if (!schemaPath) {
-    throw new Error(`Could not find ${schemaPathFromArgs || 'schema.prisma'}`)
+    throw new Error(
+      `Either provide ${chalk.greenBright('--schema')} ${chalk.bold(
+        'or',
+      )} configure a path in your package.json in a \`prisma.schema\` field ${chalk.bold(
+        'or',
+      )} make sure that you are in a folder with a ${chalk.greenBright(
+        'schema.prisma',
+      )} file.`,
+    )
```

Comment:
The message here is a bit different and missing the link https://pris.ly/d/prisma-schema-location`, is that expected?

Reply:
Thanks for catching it 🙏. Nah, that's what happens when you change 3 times 10+ error messages in different locations 🙈